### PR TITLE
Click in header not loading properties panel

### PR DESCRIPTION
**Pull request by danielzhe**
When a header of ContextExpression or the output column in DecisionTale are selected the properties panel was not being loaded.

### CODE REVIEW

File: packages/boxed-expression-component/src/expressions/ContextExpression/ContextExpression.tsx [hunk 1]
```diff
@@ -118,7 +118,7 @@ export function ContextExpression(contextExpression: ContextExpressionDefinition
   const beeTableColumns = useMemo<ReactTable.Column<ROWTYPE>[]>(() => {
     return [
       {
-        accessor: "context-expression" as any, // FIXME: Tiago -> ?
+        accessor: contextExpression.id as any, // FIXME: Tiago -> ?
```
Comment (danielzhe) [inline packages/boxed-expression-component/src/expressions/ContextExpression/ContextExpression.tsx:121]: People from the future will fix the issue about "accessor" not being correctly typed as string.

**Comment by danielzhe** [comment]
@yesamer Can you please take a look? Is the issue you reported in private chat.

File: packages/boxed-expression-component/src/expressions/ContextExpression/ContextExpression.tsx (hunk 1 above)
Comment (tiagobento) [reply packages/boxed-expression-component/src/expressions/ContextExpression/ContextExpression.tsx:121]: Agreed :)

**Review by tiagobento — APPROVED** [review]

**Comment by yesamer** [comment]
Additional cases to cover:

1: Literal header


https://user-images.githubusercontent.com/16005046/230318365-98893bc0-f376-4667-a607-4a8111edad49.mov

**Comment by yesamer** [comment]
2: An empty expression should show an empty property panel (like in the old editor)

https://user-images.githubusercontent.com/16005046/230319369-0a640ef8-19b0-430b-90df-86d20f749b1b.mov

**Comment by yesamer** [comment]
3: **Function** header


Uploading Screen Recording 2023-04-06 at 10.26.50.mov…

**Comment by yesamer** [comment]
4. **Invocation** Header

https://user-images.githubusercontent.com/16005046/230321061-fff97c9b-3adb-4e2e-a20b-e8babbca3e09.mov

**Comment by danielzhe** [comment]
@yesamer Changes applied!

**Comment by yesamer** [comment]
@danielzhe Thank you, most of the points have been resolved. Point 2. is still present. 

I found an additional case we forgot to manage: BKM nodes:


https://user-images.githubusercontent.com/16005046/231087984-f1b74ddb-ce4f-49d2-bf3b-7a548cd49658.mov

Thank you!

**Comment by danielzhe** [comment]
@yesamer Changes applied!

**Comment by yesamer** [comment]
@danielzhe In some scenarios is still reproducing: 

https://user-images.githubusercontent.com/16005046/231396257-0ed68462-75ee-4174-9d65-69c9b67d7cd3.mov

**Review by tiagobento — APPROVED** [review]

**Pull request merged**